neurogym/config/config.py

```diff
@@ -103,6 +122,5 @@ def expand_env_vars(cls, value: str) -> str:
         return value
 
 
-# Default configuration object
-# ==================================================
+# Shared singleton config used throughout the project
 config = Config()
```

Comment:
This would be impossible if there are no default values assigned to every config variable.

Reply:
Which default values are missing?